grpc/internal/status/status.go

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
Please prepend the following header to make CI happy
```go
/*
   Copyright The containerd Authors.
   Licensed under the Apache License, Version 2.0 (the "License");
   you may not use this file except in compliance with the License.
   You may obtain a copy of the License at
       http://www.apache.org/licenses/LICENSE-2.0
   Unless required by applicable law or agreed to in writing, software
   distributed under the License is distributed on an "AS IS" BASIS,
   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   See the License for the specific language governing permissions and
   limitations under the License.
*/
```

Reply:
Hi Akihiro,
I copied the code from the google grpc original code, so I wanted to keep the original "Copyright 2020 gRPC authors" line. I will leave the original licence header and add the new one on top. Thanks for the advice!